src/main/io/bf_osd_symbols.h

```diff
@@ -32,12 +32,14 @@
 #define BF_SYM_ROLL                    0x14
 #define BF_SYM_PITCH                   0x15
 #define BF_SYM_TEMPERATURE             0x7A
+#define BF_SYM_MAX                     0x24
+#define BF_SYM_PROFILE                 0x7C
 
 // GPS and navigation
 #define BF_SYM_LAT                     0x89
 #define BF_SYM_LON                     0x98
 #define BF_SYM_ALTITUDE                0x7F
-#define BF_SYM_TOTAL_DISTANCE          0x71
+#define BF_SYM_TOTAL_DISTANCE          0x2A
```

Comment:
This is a bug in DJI's font. I don't think we should change this.

In theory they are working with the BF team and may fix this.
I don't think we should deviate.

Reply:
I have tested and they are working on my goggles. If DJI changes it, we change it as well, I don't see a problem here.